user-guide-chapters/introduction/why.adoc

```diff
@@ -65,3 +65,8 @@ and the possible attack surface.
 [14] https://github.com/notaryproject/notation
 
 [15] https://github.com/sigstore/cosign
+
```

Comment:
lets add a paragraph highlighting the OCI conformance, minimal deployment using single binary as the core strength of zot. built for developers by developers, batteries included etc.

I would like to have the table with only Zot features and not have comparison to ANY other registries. Also lets remove these references because we are the only one in the table.

Reply:
We can still call out key differentiators in terms of criteria because that is what readers will be thinking.

Reply:
Added highlights paragraph, removed Docker column and references.